demo/index.html

```diff
@@ -86,26 +87,26 @@
     <div class="vertical-section">
       <paper-input label="label" char-counter></paper-input>
 
-      <paper-input label="at most 10 letters" char-counter auto-validate pattern="[a-zA-Z]*" maxlength="10" error-message="letters only!"></paper-input>
+      <paper-input label="at most 10 letters" char-counter auto-validate pattern="[a-zA-Z]*" maxlength="10" error-message="letters only!" tabindex="0"></paper-input>
 
-      <paper-textarea label="textarea" char-counter></paper-textarea>
+      <paper-textarea label="textarea" tabindex="0" char-counter></paper-textarea>
 
-      <paper-textarea label="textarea with maxlength" char-counter maxlength="10"></paper-textarea>
+      <paper-textarea label="textarea with maxlength" tabindex="0" maxlength="10" char-counter></paper-textarea>
 
-      <paper-textarea label="text area with rows and max-rows" rows="3" max-rows="4"></paper-textarea>
+      <paper-textarea label="text area with rows and max-rows" rows="3" max-rows="4" tabindex="0"></paper-textarea>
     </div>
 
     <h4>Prefixes and Suffixes</h4>
     <div class="vertical-section">
-      <paper-input class="short" label="total" type="number">
+      <paper-input class="short" label="total" type="number" tabindex="0">
         <div prefix>$</div>
       </paper-input>
 
-      <paper-input class="short" label="username">
+      <paper-input class="short" label="username" tabindex="0">
         <div suffix>@email.com</div>
       </paper-input>
 
-      <paper-input class="short" label="icons and buttons" id="inputWithButton">
+      <paper-input class="short" label="icons and buttons" tabindex="0" id="inputWithButton">
```

Comment:
If everything needs a tabindex, maybe this should be the default in `PaperInputBehavior`?